BeatSinger/Helpers/Extensions.cs

```diff
@@ -52,5 +52,14 @@ public static JSONArray ToJsonArray(this IEnumerable<Subtitle> subtitles)
             }
             return array;
         }
+        public static JSONObject ToJson(this SubtitleContainer subtitles)
```

Comment:
Here and in some other places: please separate methods with an empty line.

(Also in other places but the GitHub code viewer doesn't make it easy to see them: please make sure all indentation is made with spaces instead of tabs, and please make sure all files end with a newline)